test/unit/io/structure_file/format_vienna_test.cpp

```diff
@@ -55,16 +55,16 @@ struct read : public ::testing::Test
         { "example 2" }
     };
 
-    std::vector<rna4_vector> const expected_seq
+    std::vector<rna5_vector> const expected_seq
     {
-        { "GCGGAUUUAGCUCAGUUGGGAGAGCGCCAGACUGAAGAUUUGGAGGUCCUGUGUUCGAUCCACAGAAUUCGCA"_rna4 },
-        { "UUGGAGUACACAACCUGUACACUCUUUC"_rna4 }
+        { "GCGGAUUUAGCUCAGUUGGGAGAGCGCCAGACUGAAGAUUUGGAGGUCCUGUGUUCGAUCCACAGAAUUCGCA"_rna5 },
+        { "UUGGAGUACACAACCUGUACACUCUUUC"_rna5 }
     };
 
-    std::vector<std::vector<dot_bracket3>> const expected_structure
+    std::vector<std::vector<wuss<51>>> const expected_structure
```

Comment:
Oo such a fundamental change? That was copy'n'paste error before?

Reply:
`wuss<51>` is the default structure alphabet if you use the public interface. The test would be much more bloated if I would add a traits object that changes the default alphabet to `dot_bracket` because automatic type deduction would not kick in if I specialise the file with its first template parameter. So I thought, since we do not test several alphabets yet and it should (and does) work with either alphabet, I might change it for simplicity.